notebooks/15_learning_a_decision_tree/index.md

```diff
@@ -0,0 +1,338 @@
+# Learning a Decision Tree (Classification)
+
```

Comment:
Well done! You did a great job building up this LN!

A few points to consider:
- Double-check the English texts for the correct use of punctuations. (e.g. after "therefore" a "," should be put. Lots of these misuses exist in your text. consider using Grammarly or other similar platforms for convenience)
- In the reference section, you are better off putting hyperlinks rather than raw links.
- Most equations and tables would look better if they were centered.
- Justifying the format of your paragraphs could also help with the improvement of the LN's readability.

Reply:
@aliash98

Reply:
@aliash98